content/en/blog/_posts/2024-04-01-validating-admission-policy-ga/index.md

```diff
@@ -0,0 +1,216 @@
+------
+layout: blog
+title: "Kubernetes 1.30: Validating Admission Policy Is Generally Available"
+slug: validating-admission-policy-ga
+date: 2024-04-01
```

Comment:
```suggestion
date: 2024-04-24
```
The Comms Team has scheduled a publication date for this post for April 24, 2024. Please update the date on the PR, thanks!